uint8 seq
float64 gimbal_base_yaw
float64 gimbal_map_yaw